sale_order_line_price_history/__manifest__.py

```diff
@@ -2,14 +2,15 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale order line price history",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Sales Management",
     "author": "Tecnativa, "
               "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow/",
+    "summary": "Sale order line price history",
```

Comment:
Don't repeat the same as the name. It's not required and I prefer to not put it.

Reply:
@pedrobaeza Ok, I will remove it.